app/controllers/registrations_controller.rb

```diff
@@ -13,6 +14,14 @@ def new
 
   private
 
```

Comment:
Looks good! Such a minor nitpick here but I'd prefer a method named `set_layout` over `choose_layout` but that's all.